### PR TITLE
pickle: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pickle.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pickle
+
+> A PHP extension installer based on Composer.
+> More information: <https://github.com/FriendsOfPHP/pickle>.
+
+- Install a specific PHP extension:
+
+`pickle install {{memcache}}`
```
Comment (andrik) [inline pages/common/pickle.md]:
```suggestion
`pickle install {{extension_name}}`
```

Comment (pxgamer) [reply pages/common/pickle.md]: I put `memcache` as a definitive example. But it does look a bit odd. 🤔

In PHP, extension names can be referenced as:
- `memcache`
- `php-memcache` (won't work here, but used in package managers)
- `ext-memcache` (won't work here, but used in Composer)

So I thought having an example would be clearer.

Comment (mebeim) [reply pages/common/pickle.md]: Maybe something like this? :thinking:

    - Install a specific PHP extension (don't include `php-` or `ext-` in the name):

    `pickle install {{extension_name}}`

Comment (agnivade) [reply pages/common/pickle.md]: Using a concrete name definitely encouraged and in the spirit of tldr pages. I don't see any issues with it.

File: pages/common/pickle.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Convert an existing PECL extension configuration to a Pickle configuration file:
+
+`pickle convert {{path/to/directory}}`
+
+- Validate a PECL extension:
+
+`pickle validate {{path/to/directory}}`
+
+- Package a PECL extension for releasing:
```
Comment (agnivade) [inline pages/common/pickle.md]: "for release" sounds better to me. But not sure if the other one is objectively correct.

Comment (pxgamer) [reply pages/common/pickle.md]: `for release` sounds better to me too, so I think I'll go with that :+1:

**Comment by agnivade** [comment]
Hmm .. that's a good point @andrik. Traditionally, our concrete examples have been numbers, IPs, mount locations etc. And for website names, we chose `example.com` to avoid preference to any specific site. In this case, it does seem that we are unnecessarily choosing `memcache`. Maybe it is good to be generic after all.

I didn't give much thought on this initially. But on a deeper look, I would have to agree to @andrik's point.

**Comment by mebeim** [comment]
Right, I too agree with @andrik .

**Pull request merged**